Adds notifications to chat for deployment actions

mcanoy opened the pull request:
Added the ability to notify to chat during each action of a release. Each notification will be threaded to the release.

Also: 
- updates the e2e notification to be a simple message and be sent to an e2e thread 
- adds the `@` to the user's handle in github comments.

After merge, the notification url should be updated to message the lodestar chat room

oybed reviewed (approved):
LGTM

jacobsee commented on .github/workflows/cancel.yml:
```diff
@@ -23,4 +25,8 @@ jobs:
+    - name: Notify chat
+      run: |
+       echo "Thread key GIIHUB_REPOSITORY-${{ steps.env_info.outputs.issuenm }} requestor ${{ steps.env_info.outputs.requestor }}"
+        [ ! -z "$URL" ] && curl "$URL&threadKey=$GIIHUB_REPOSITORY-${{ steps.env_info.outputs.issuenm}}" -H "Content-Type: application/json" -d '{ "text" : "\uD83E\uDDEF cancelling rollout as requested by ${{ steps.env_info.outputs.requestor }}" }'
```
`GIIHUB` here looks like a typo (same in the other workflow files) but otherwise looks good and the idea sounds really useful 👍

mcanoy replied:
nice catch - didn't effect the functionality atm but was unintended and could effect future threading. I'd like to blame this butterfly keyboard but that doesn't account for the missing `T`

oybed commented:
Posted in internal chat, but adding here as well just in case:
One quick question - would we want the option to perhaps set different notification URLs for e2e v.s. release? I suspect the e2e traffic may spam our main channel if we wanted to point the notifications to here.

mcanoy commented:
Updated incorporating feedback. Added support for different notifications. For the timing being they will point to the same room. I've added them to the secrets so a merge will work. Please delete **NOTIFICATION_URL** secret after merging

oybed reviewed (approved):
LGTM
